server/src/main/java/io/crate/metadata/pgcatalog/PgCatalogTableDefinitions.java

```diff
@@ -81,7 +84,8 @@ public PgCatalogTableDefinitions(InformationSchemaIterables informationSchemaIte
             false));
         tableDefinitions.put(PgNamespaceTable.IDENT, new StaticTableDefinition<>(
             informationSchemaIterables::schemas,
-            (user, s) -> user.hasAnyPrivilege(Privilege.Clazz.SCHEMA, s.name()),
+            (user, s) -> user.hasAnyPrivilege(Privilege.Clazz.SCHEMA, s.name())
+                         || isUnderSystemSchemas(s.name()),
```

Comment:
Looking at the usage of `isUnderSystemSchemas` it looks like the schema name alone would also always be available. So instead of using `fqn` we could check on `schema()`.

Avoids doing some work (`fqn()` concats a string).